index.js

```diff
@@ -267,4 +263,13 @@ Emailer.admin = {
     }
 };
 
+Emailer.addMiddleware = function(data, callback) {
+    data.app.head('/emailer-mandrill/reply', function(req, res) {
+        res.sendStatus(200);
+    });
+    data.app.post('/emailer-mandrill/reply', Emailer.receive);
+
+    callback();
+};
+
```

Comment:
Alright. Just tested it again and your recommendation still seemed to work! That means the only fix left in this PR will be the URL that it shows on the admin page.

I will move the GitHub-like filtering to a separate PR.